release-notes/action.yml

```diff
@@ -0,0 +1,32 @@
+name: "Release Notes"
+description: "Captures release notes from a release and pushes them to a database in Notion"
+
+inputs:
+  notion-token:
+    description: "Token for Notion API"
+    required: true
+  notion-database-id:
+    description: "Database ID to add pages to"
+    required: true
+  product:
+    description: "The product name"
+    required: true
+  release-tag:
+    description: "The release tag"
+    required: true
+  release-notes:
+    description: "The release notes from the release body"
+    required: true
+
+runs:
+  using: composite
+  steps:
```

Comment:
if you're only doing one step here, I think you can omit this and just use the docker action?

Reply:
I left it composite in case we wanted to do any other actions (post to Slack, etc). Also, this is my first composite action, can you suggest a change?

Reply:
that's fine, I wasn't exactly sure why you were making it like that but now it makes sense!